src/test/java/org/apache/brooklyn/cloudfoundry/entity/VanillaPaasApplicationCloudFoundryDriverTest.java

```diff
@@ -52,7 +54,6 @@
 import com.squareup.okhttp.mockwebserver.RecordedRequest;
 
 public class VanillaPaasApplicationCloudFoundryDriverTest extends AbstractCloudFoundryUnitTest {
```

Comment:
I think an integration test with `FakeCloudFoundryClient` makes sense.
You can keep the unit test as well